3DAmsterdam/Assets/Amsterdam3D/Scripts/Interface/Search/SearchResult.cs

```diff
@@ -75,6 +75,9 @@ IEnumerator FindLocationByIDLookup()
 
                     ParentList.ShowResultsList(false);
                     CameraModeChanger.Instance.CurrentCameraControls.MoveAndFocusOnLocation(targetLocation, new Quaternion());
+
+                    print(jsonStringResult);
+                    FindObjectOfType<SelectByID>().HighlightSelectedID(lookupData.response.docs[0].nummeraanduiding_id);
```

Comment:
Niet beter om van tevoren al FindObjectOfType te doen ipv in deze coroutine?

Reply:
Agreed!